packages/block-editor/src/components/block-controls/hook.js

```diff
@@ -6,14 +6,15 @@
  * Internal dependencies
  */
 import groups from './groups';
-import useDisplayBlockControls from '../use-display-block-controls';
+import { useBlockEditContext } from '../block-edit/context';
 
 export default function useBlockControlsFill( group, shareWithChildBlocks ) {
-	const { isDisplayed, isParentDisplayed } = useDisplayBlockControls();
-	if ( isDisplayed ) {
+	const { mayDisplayControls, mayDisplayParentControls } =
```

Comment:
So these are two new APIs?

Reply:
This is internal context right?

Reply:
Oh, I guess it's not... We could lock it with symbols I guess?

Reply:
Fixed it 😊